src/main/java/com/gooddata/executeafm/resultspec/Aggregation.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+package com.gooddata.executeafm.resultspec;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum Aggregation {
```

Comment:
javadoc would be nice
and isn't it possible to reuse `com.gooddata.executeafm.afm.Aggregation`?